checkout-ui-custom/src/_js/_v.custom.checkout.ui.js

```diff
@@ -47,7 +47,11 @@ class checkoutCustom {
 
   builder() {
     const _this = this
-
+    if ($('.cart-more-options').length === 0) {
+      return setTimeout(() => {
+        _this.builder()
```

Comment:
This is not good, man! `builder()` is calling `builder()` after 1.5s and `ajaxComplete` calls builder(). There's not even a debounce.

Reply:
No problem, the builder function needs `$('.cart-more-options')` existing, `ajaxComplete` executes the `init` function and `init` executes `builder` then builder is calling builder only if `$('.cart-more-options')` exists